domain/transfer.go

```diff
@@ -19,8 +20,6 @@ type transferProcess struct {
 // TransferProcessHandler manages the process of transferring funds between
 // accounts.
 type TransferProcessHandler struct {
-	dogma.NoTimeoutMessagesBehavior
-	dogma.NoTimeoutHintBehavior
```

Comment:
`NoTimeoutHintBehavior` is unrelated to "timeout messages" and can stay.  It just provides the default implementation of the `TimeoutHint()` method.

Reply:
Re-added `NoTimeoutHintBehavior` and removed `TimeoutHint()` function.